README.md

```diff
@@ -18,6 +18,14 @@ plugins {
 apply plugin: 'org.checkerframework'
 ```
 
+Note that the `org.checkerframework` plugin
+must be applied *after* whatever
+plugins introduce your Java compilation tasks (usually the `java` or `java-library`
+plugin for non-Android builds). If you apply that plugin 
```

Comment:
How is the second sentence different than the first?  They seem to contain the same information.

A few lines above is "Add the following to your `build.gradle` file:" and a single block of code.  If that block of code should not be inserted as is, please break it into two blocks, possibly with brief explanatory text about each one.